### DISCUSSION TITLE
🔴⭕Update for all - Step 1⭕🔴

### DISCUSSION THREAD

**Discussion by SarthakKeshari** [SarthakKeshari/Java-Questions-and-Solutions#56] [Ideas]
***
# Hello team, 👋
### Your amazing efforts can be seen from very first day and this is really motivating and creates positive project building spirit.
***

### 🌄Opening a new way into this project to make it more user-friendly and to start this I wish to have your suggestions. #Letsthinktogether🤩

<br>

### I am elated to call upon everyone to suggest a name for the website (***only one name per participant***). We'll collect the ideas for name till 2AM(03/10/2021)

- The participant whose suggested name gets selected will get the chance to contribute a word file to the repository stating the website name and the reason behind selecting that name.
- The selected participant will be given **level 2 tag as well as hacktoberfest-accepted tag** for this contribution.

#️⃣Winning #️⃣EverySuggestionMatters

**Comment by Lakshay-Pruthi** [comment]
@SarthakKeshari  I know it is bit late but if you consider my recommendation then it is  OpenCode 
reason : an open source free website for coding enthusiasts containing data like which question asked in which company free for everyone

**Reply by SarthakKeshari** [reply]
Issue No.193 has been selected for this. But if the issues crosses the deadline. You make take over. By reclaiming the issue.
